app/controllers/concerns/remember_device_concern.rb

```diff
@@ -72,12 +72,6 @@ def handle_valid_remember_device_cookie
 
   def handle_valid_remember_device_analytics
     analytics.track_event(Analytics::REMEMBERED_DEVICE_USED_FOR_AUTH, {})
-    GoogleAnalyticsMeasurement.new(
```

Comment:
we have some server-side google analytics, but all are paired with a call to our own analytics, so we shouldn't be missing any events by removing it